### DISCUSSION TITLE
Curso Fullstack.NET - Autenticação via cookie

### DISCUSSION THREAD

**Discussion by aps180180** [balta-io/forum#616] [Tira Dúvidas] [answered]
Olá @andrebaltieri  estou concluindo o curso e achei fantástico!
Gostaria de utilizar a autenticação JWT no lugar da via cookies.
Poderia me orientar quanto as mudanças necessárias?
Grato!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @aps180180, como vai?

Claro, aqui está a documentação oficial:
https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-8.0#use-token-based-authentication
